src/assets/style.css

```diff
@@ -1,12 +1,17 @@
+header {
+  word-wrap:break-word;
```

Comment:
In which cases is this necessary? I thought we added the `&shy;` exactly to avoid breaking mid-word. 🤔